me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/listener/IDmeGetAuthCodeListener.java

```diff
@@ -0,0 +1,9 @@
+package me.id.webverifylib.listener;
+
+/**
+ * Listener to get the auth token
+ */
+public interface IDmeGetAuthCodeListener {
```

Comment:
I think you should consider adding generic classes for this, as RxJava, Retrofit and other libraries does with `Action`, `Function`, `Callback` and the like.